lms/djangoapps/discussion_api/api.py

```diff
@@ -335,7 +338,10 @@ def get_thread_list(
         paginated_results = context["cc_requester"].subscribed_threads(query_params)
     else:
         query_params["course_id"] = unicode(course.id)
-        query_params["commentable_ids"] = ",".join(topic_id_list) if topic_id_list else None
+        if inline_discussion_id:
+            query_params["commentable_id"] = inline_discussion_id
```

Comment:
Does the comments service already allow filtering by discussion_id?  Or will there be a related PR that updates the comments service?